setup-system.sh

```diff
@@ -43,8 +43,16 @@ function install_sublime_3() {
     sudo dpkg -i /tmp/sublime-text_build-3065_amd64.deb
 }
 
+# Download and install atom as best we know how
+function install_atom() {
+    echo "~ Downloading atom deb package"
+    sudo wget -P /tmp https://atom-installer.github.com/v1.5.3/atom-amd64.deb
```

Comment:
Could we make this `wget -O /tmp/atom.deb https://atom.io/download/deb`? That way it'll track the latest version. Also the `wget` part shoudn't need sudo - anyone can use the `/tmp` directory.